docs/docsite/rst/reference_appendices/release_and_maintenance.rst

```diff
@@ -1,86 +1,94 @@
+Developed and maintained in Collection repositories   Developed and maintained in ansible/ansible repository
+====================================================  ========================================================
+
+Many community users install the Ansible community package. The Ansible community package offers the functionality that existed in Ansible 2.9, with more than 85 Collections containing thousands of modules and plugins. The ``ansible-core`` option is primarily for developers and users who want to install only the collections they need.
 
 .. contents::
    :local:
 
 .. _release_cycle:
 
-Ansible release cycle
------------------------
+Release cycle overview
+======================
 
-Ansible is developed and released on a flexible release cycle.
-This cycle can be extended in order to allow for larger changes to be properly
-implemented and tested before a new release is made available. See :ref:`roadmaps` for upcoming release details.
+The two community releases are related - the release cycle follows this pattern:
 
-For Ansible version 2.10 or later, the major release is maintained for one release cycle. When the next release comes out (for example, 2.11), the older release (2.10 in this example) is no longer maintained.
+#. Release of a new ansible-core major version, for example, ansible-core 2.11
 
-If you are using a release of Ansible that is no longer maintained, we strongly
-encourage you to upgrade as soon as possible in order to benefit from the
-latest features and security fixes.
+   * New release of ansible-core and two prior versions are now maintained (in this case, ansible-base 2.10, Ansible 2.9)
+   * Work on new features for ansible-core continues in the ``devel`` branch
 
-Older, unmaintained versions of Ansible can contain unfixed security
-vulnerabilities (*CVE*).
+#. Collection freeze (no new Collections or new versions of existing Collections) on the Ansible community package
+#. Release candidate for Ansible community package, testing, additional release candidates as necessary
+#. Release of a new Ansible community package major version based on the new ansible-core, for example, Ansible 4.0.0 based on ansible-core 2.11
 
-You can refer to the :ref:`porting guides<porting_guides>` for tips on updating your Ansible
-playbooks to run on newer versions. For Ansible 2.10 and later releases, you can install the Ansible package with ``pip``. See :ref:`intro_installation_guide` for details.  For older releases, You can download the Ansible release from `<https://releases.ansible.com/ansible/>`_.
+   * Newest release of the Ansible community package is the only version now maintained
+   * Work on new features continues in Collections
+   * Individual Collections can make multiple minor and/or major releases
+
+#. Minor releases of three maintained ansible-core versions every three weeks (2.11.1)
+#. Minor releases of the single maintained Ansible community package version every three weeks (4.1.0)
```

Comment:
Are we timing Ansible package minor release to always pick up the ansible-core minor release? As in can a user assume Ansible 4.x.0 will have ansible-core 2.11.x ? (one to one correspondence?) or is it too tricky to commit to syncing up the minor releases like this?

Reply:
We haven't committed to updating the included version of ansible-core on a specific schedule, so I think we shouldn't mention anything specific here.